rclcpp/include/rclcpp/node.hpp

```diff
@@ -103,9 +103,11 @@ class Node : public std::enable_shared_from_this<Node>
     rclcpp::Context::SharedPtr context,
     const std::vector<std::string> & arguments,
     const std::vector<Parameter> & initial_parameters,
+    const rmw_qos_profile_t &qos_profile,
```

Comment:
What is the issue with the default QoS profile for the parameter event publisher? (I'm trying to understand the motivation behind making it configurable).

Reply:
(see: https://github.com/ros2/rclcpp/issues/591) Basically it defaults to `KEEP_ALL` (which could be undesirable in certain systems) and ideally should be configurable by the user if desired. The better question in my opinion is "why should it not be configurable" :)

Reply:
> I would rename the parameter so it is clear that the profile is for the event publisher, e.g. `parameter_event_pub_qos_profile`. And document it in the API above.

👍